resources/scripts.js

```diff
@@ -125,112 +188,354 @@ function buttonPosition(loc) {
+    comeAmount = parseInt(comeAmount);
+    if (validateBet(comeAmount) === false) {
+        return;
+    }
+    lastPlayerMoney = playerMoney;
+    playerMoney -= comeAmount;
+    secondaryBetList.push(new SecondaryBet(1, comeAmount));
+    secondaryBetListO[1] = comeAmount;
+    moneyChange(comeAmount);
+    betDialogElement.disabled = true;
+    comeButtonElement.disabled = true;
+    return;
+}
+
+function makePlace(num) {
+    placeDialogAmount.setAttribute('max', playerMoney);
+    makePlaceNumClicked = num;
+    placeDialog.showModal();
+    return;
+}
+
+class SecondaryBet {
+    constructor(point, amount) {
+        this.point = point;
+        this.amount = amount;
+    }
+}
+
+function removeBet(index) {
+    if (index === 0) {
+        secondaryBetList = secondaryBetList.slice(1);
+    } else if (index === secondaryBetList.length) {
+        secondaryBetList = secondaryBetList.slice(0, -1);
+    } else {
+        const halfBefore = secondaryBetList.slice(0, index);
+        const halfAfter = secondaryBetList.slice(index + 1);
+        secondaryBetList = halfBefore.concat(halfAfter);
+    }
+    return;
+}
+
+function pushSecondaryBets() {
+    if (secondaryBetList.length !== 0) {
+        for (let i = 0; i < secondaryBetList.length; i++) {
+            playerMoney += secondaryBetList[i].amount;
+        }
+        for (let i = 0; i < secondaryBetList.length; i++) {
+            removeBet(i);
+        }
+    }
     return;
 }
 
 function gameOver() {
-    displayMessage(`You are bankrupt! Please choose 'New Game' from the menu to play again`);
-    buttonPosition(1);
-    rollButtonState(false);
-    betButtonState(false);
+    displayMessage(BANKRUPT);
+    playerRoll = 1;
+    buttonPosition(playerRoll);
+    buttonStates();
+    return;
+}
+
+async function moneyChange(newBet) {
+    totalBets += newBet;
+    playerMoneyElement.innerHTML = playerMoney;
+    playerBetElement.innerHTML = totalBets;
+    if (playerMoney === lastPlayerMoney) {
+        return;
+    }
+    if (playerMoney < lastPlayerMoney) {
+        playerMoneyElement.style.transition = 'all 0.5s';
+        playerMoneyElement.style.fontSize = '75%';
+        playerMoneyElement.style.color = '#ff0000';
+        await new Promise(resolve => setTimeout(resolve, 500));
+        playerMoneyElement.style.transition = 'all 0.5s';
+        playerMoneyElement.style.fontSize = '120%';
+        playerMoneyElement.style.color = '#00ff00';
+        return;
+    }
+    if (playerMoney > lastPlayerMoney) {
+        playerMoneyElement.style.transition = 'all 0.5s';
+        playerMoneyElement.style.fontSize = '175%';
+        playerMoneyElement.style.color = '#0000ff';
+        await new Promise(resolve => setTimeout(resolve, 500));
+        playerMoneyElement.style.transition = 'all 0.5s';
+        playerMoneyElement.style.fontSize = '120%';
+        playerMoneyElement.style.color = '#00ff00';
+        return;
+    }
+    return;
 }
 
 function displayMessage(str) {
```

Comment:
Function should be refactored to include a roll summary (created when a secondary roll happens).

Reply:
As this is outside the scope of this PR, I would like to merge and start a new PR which encompasses:
1. Displaying messages to the user
2. Displaying bets to the user in a meaningful way